### PR TITLE
Better kiwix-manage usage()

### LINKED ISSUE #129: Improve the 'usage' information for kiwix-manage
The current 'usage' information for several of the kiwix tools isn't sufficiently clear to enable a novice to use them. The two tools I tried are `kiwix-install` and `kiwix-manage`, the others may also benefit from improvements to the usage statements they emit when run.

After a working session with @kelson42 I now have a better idea of what usage would be valuable so I'm happy to try improving the information emitted by these utilities. It'd also be healthy to revise the documentation on the Kiwix wiki, again I'm happy to do so.

### PR DESCRIPTION
**Pull request by kelson42**
Fix #129

### CODE REVIEW

File: src/manager/kiwix-manage.cpp [hunk 1]
```diff
@@ -56,20 +56,43 @@ void show(kiwix::Library* library, const std::string& bookId)
...
+            << "\t\t\t* add: add a ZIM file to the library" << std::endl
+            << "\t\t\t* remove: remove a ZIM file from the library" << std::endl 
+            << "\t\t\t* show: show the content of the library"
+	    << std::endl << std::endl
+            << "\tZIM_ID\t\tZIM file unique ID"
+            << std::endl << std::endl
+            << "\tOPTIONS\t\tCustom options for \"add\" action:" << std::endl
+            << "\t\t\t--zimPathToSave=CUSTOM_ZIM_PATH to replace the current ZIM file path" << std::endl
+            << "\t\t\t--url=HTTP_ZIM_URL to create an \"url\" attribute for the online version of the ZIM file" << std::endl
+            << "\t\t\t--current to specify the ZIM to open per default in this library" << std::endl
```
Comment (mgautierfr) [inline src/manager/kiwix-manage.cpp]: There is no current option.
It has been removed 6 months ago (October 2018). I should have remove it from the usage string also.

Comment (kelson42) [reply src/manager/kiwix-manage.cpp]: @mgautierfr good, will remove it and merge. Thx.

File: src/manager/kiwix-manage.cpp [hunk 2]
```diff
@@ -56,20 +56,43 @@ void show(kiwix::Library* library, const std::string& bookId)
...
-       << endl;
-  cerr << "\tkiwix-manage LIBRARY_PATH remove CONTENTID1 [CONTENTID2]" << endl;
+  std::cout << "Usage:" << std::endl
+            << "\tkiwix-manage LIBRARY_PATH add ZIM_PATH [OPTIONS]" << std::endl
+            << "\tkiwix-manage LIBRARY_PATH remove ZIM_ID [ZIM_ID]..." << std::endl
+	    << "\tkiwix-manage LIBRARY_PATH show [ZIM_ID]..." << std::endl
+            << std::endl
+    
+            << "Purpose:" << std::endl
+            << "\tManipulating Kiwix library XML file"
```
Comment (julianharty) [inline src/manager/kiwix-manage.cpp]:
```suggestion
            << "\tManipulates the Kiwix library XML file"
            << "\twhich enables various Kiwix software to offer ZIM content to users. See https://wiki.kiwix.org/wiki/Library" 
```

Comment (kelson42) [reply src/manager/kiwix-manage.cpp]: @julianharty I have added a few examples

File: src/manager/kiwix-manage.cpp (hunk 1 above)
Comment (kelson42) [reply src/manager/kiwix-manage.cpp]: @mgautierfr removed

File: src/manager/kiwix-manage.cpp [hunk 3]
```diff
@@ -52,20 +52,48 @@ void show(kiwix::Library* library, const std::string& bookId)
...
+            << "\tZIM_ID\t\tZIM file unique ID"
+            << std::endl << std::endl
+            << "\tOPTIONS\t\tCustom options for \"add\" action:" << std::endl
+            << "\t\t\t--zimPathToSave=CUSTOM_ZIM_PATH to replace the current ZIM file path" << std::endl
+            << "\t\t\t--url=HTTP_ZIM_URL to create an \"url\" attribute for the online version of the ZIM file" << std::endl
+            << std::endl
+
+            << "Examples:" << std::endl
+            << "\tAdd ZIM files to library: kiwix-manage my_library.xml add first.zim second.zim" << std::endl
+            << "\tRemove ZIM fiels from library: kiwix-manage my_library.xml remove e5c2c003-b49e-2756-5176-5d9c86393dd9" << std::endl
```
Comment (mgautierfr) [inline src/manager/kiwix-manage.cpp]: fiels -> files

Comment (kelson42) [reply src/manager/kiwix-manage.cpp]: Fixed in PR 284

**Pull request merged**